### PR TITLE
Juniper EX2300 CoA Port Bounce

### PR DESCRIPTION
**Pull request by lyubomirtraykov**
# Description
Perform CoA Port-Bounce on Juniper EX2300 Switch 

# Impacts
(REQUIRED)
Port Bounce method

# Delete branch after merge
YES

# Checklist
- [n/a ] Document the feature
- [ yes] Add unit tests
- [ n/a] Add acceptance tests (TestLink)

# NEWS file entries
(REQUIRED, but may be optional...)
## Enhancements
* Enhance Juniper EX2300

### CODE REVIEW

File: lib/pf/Switch/Juniper/EX2300.pm [hunk 1]
```diff
@@ -105,6 +105,72 @@ sub radiusDisconnect {
...
+    }
+
+    if (!defined($self->{'_radiusSecret'})) {
+        $logger->warn(
+            "Unable to perform RADIUS CoA-Request on $self->{'_id'}: RADIUS Shared Secret not configured"
+        );
+        return;
+    }
+
+    $logger->info("boucing $mac using RADIUS CoA-Request method");
```
Comment (julsemaan) [inline lib/pf/Switch/Juniper/EX2300.pm]: boucing -> bouncing

**Review by julsemaan — CHANGES_REQUESTED** [review]
Just a minor logging detail to fix and then its ready to merge for 9.2

**Comment by lyubomirtraykov** [comment]
Using _radiusBounceMac is better because we don't need to get the mac from the  ifIndex. Will the _radiusBounceMac be called from the GUI?

**Comment by julsemaan** [comment]
From what I can tell _radiusBounce isn't being called anywhere so you need to use setAdminStatus like in the previous version of your PR or else it won't do anything

File: lib/pf/Switch/Juniper/EX2300.pm (hunk 1 above)
Comment (julsemaan) [reply lib/pf/Switch/Juniper/EX2300.pm]: You still have boucing instead of bouncing

**Pull request merged**